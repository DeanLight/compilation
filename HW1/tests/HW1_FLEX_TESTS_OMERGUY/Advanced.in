%% My comment brings all the boys to the yard
% it's all about the % bout da % ba dam tzzzz%%%%
stream

endstream
stream

endstream
% =====================================================
%					where is my dog
% =====================================================
stream
stream
endstream

stream
stream
stream
stream
can't find him
stream
stream
endstream

stream
momma
just killed a snail
put a salt against his head
pour the flour now he's bread
endstream

stream
endstreamadafucka
endstream

(stream\
\
endstream)

(stream\n\r\nendstream)

stream
(his name is fluffy)
endstream


<20202020202020202020205f5f2e202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020200a20202020202020202e2d222e27202020202020202020202020202020202020202020202e2d2d2e2020202020202020202020205f2e2e2e5f202020200a2020202020202e27202e272020202020202020202020202020202020202020202e27202020205c202020202020202e2d222220205f5f2022222d2e200a20202020202f20202f2020202020202020202020202020202020202020202e27202020202020203a202d2d2e2e3a5f5f2e2d2222202022222d2e205c0a202020203a20203a2020202020202020202020202020202020202020202f2020202020202020203b2e642424202020207362705f2e2d22222d3a5f3a0a202020203b20203a20202020202020202020202020202020202020203a202e5f202020202020203a50202e2d2e2020202c22545020202020202020200a202020203a2020205c20202020202020202020202020202020202020205c2020542d2d2e2e2e2d3b203a2064246220203a64246220202020202020200a20202020205c202020602e202020202020202020202020202020202020205c2020602e2e27202020203b2024202420203b24202420202020202020200a202020202020602e202020222d2e2020202020202020202020202020202020292e20202020202020203a2054245020203a54245020202020202020200a20202020202020205c2e2e2d2d2d5e2e2e202020202020202020202020202f2020202020202020202020602d2720202020602e5f602e5f20202020200a202020202020202e272020202020202020222d2e202020202020202e2d222020202020202020202020202020202020202020205424242462202020200a2020202020202f20202020202020202020202020222d2e5f2e2d222020202020202020202020202020202e5f2020202020202020275e27203b2020200a20202020203a2020202020202020202020202020202020202020202020202020202020202020202020205c2e602e2020202020202020202f202020200a20202020203b20202020202020202020202020202020202020202020202020202020202020202d2e2020205c602e222d2e5f2e2d272d2720202020200a202020203a2020202020202020202020202020202020202020202020202020202020202020202e275c2020205c>
(Here is the second half :'\()
<205c205c205c2020202020202020200a202020203b20203b20202020202020202020202020202020202020202020202020202020202f3a20205c2020205c205c202e203b20202020202020200a2020203a2020203a202020202020202020202020202020202020202020202020202020202c20203b2020602e2020602e3b20203a20202020202020200a2020203b202020205c20202020202020203b2020202020202020202020202020202020202020203b20202020222d2e5f3a20203b20202020202020200a20203a202020202020602e2020202020203a2020202020202020202020202020202020202020203a2020202020202020205c2f2020202020202020200a20203b202020202020202f222d2e202020203b20202020202020202020202020202020202020203a20202020202020202020202020202020202020200a203a202020202020202f20202020222d2e203a2020202020202020202020202020202020203a203b20202020202020202020202020202020202020200a203a20202020202e272020202020202020542d3b20202020202020202020202020202020203b203b20202020202020200a203b202020203a202020202020202020203b203b202020202020202020202020202020202f20203a20202020202020200a203b202020203b202020202020202020203a203a20202020202020202020202020202e27202020203b202020202020200a3a202020203a2020202020202020202020203b3a2020202020202020205f2e2e2d225c20202020203a202020202020200a3a20202020205c20202020202020202020203a203b202020202020202f2020202020205c20202020203b2020202020200a3b202020202e20272e202020202020202020272d3b2020202020202f20202020202020203b202020203a2020202020200a3b20205c20203b203a20202020202020202020203a20202020203a2020202020202020203a20202020272d2e2020202020200a272e2e5f4c2e3a2d2720202020202020202020203a20202020203b2020202020202020203b202020202e20602e200a2020202020202020202020202020202020202020203b202020203a202020202020202020203a20205c20203b203a20200a2020202020202020202020202020202020202020203a20202020272d2e2e20202020202020272e2e5f4c2e3a2d2720200a202020202020202020202020202020202020202020203b20202020202c20602e202020202020202020202020202020200a202020202020202020202020202020202020202020203a2020205c20203b203a202020202020202020202020202020200a20202020202020202020202020202020202020202020272e2e5f5f4c2e3a2d27>

(Found him. Do you know why ? because)
<0a205f5f2020205f5f2e5f5f5f20205f5f5f2e202020202e5f5f5f5f5f5f20202020202020205f5f5f2020202e5f5f5f5f5f5f5f5f5f5f5f2e2e5f5f5f20205f5f5f2e2020202020205f5f5f2020202020202e5f5f2020205f5f2e200a7c20207c20285f20292020205c2f2020207c202020207c2020205f20205c2020202020202f2020205c20207c20202020202020202020207c7c2020205c2f2020207c20202020202f2020205c20202020207c20205c207c20207c200a7c20207c20207c2f7c20205c20202f20207c202020207c20207c5f2920207c202020202f20205e20205c20602d2d2d7c20207c2d2d2d2d607c20205c20202f20207c202020202f20205e20205c202020207c2020205c7c20207c200a7c20207c202020207c20207c5c2f7c20207c202020207c2020205f20203c202020202f20202f5f5c20205c202020207c20207c20202020207c20207c5c2f7c20207c2020202f20202f5f5c20205c2020207c20202e206020207c200a7c20207c202020207c20207c20207c20207c202020207c20207c5f2920207c20202f20205f5f5f5f5f20205c2020207c20207c20202020207c20207c20207c20207c20202f20205f5f5f5f5f20205c20207c20207c5c2020207c200a7c5f5f7c202020207c5f5f7c20207c5f5f7c202020207c5f5f5f5f5f5f2f20202f5f5f2f20202020205c5f5f5c20207c5f5f7c20202020207c5f5f7c20207c5f5f7c202f5f5f2f20202020205c5f5f5c207c5f5f7c205c5f5f7c0a>

<4
8 65 6 c 6 c 6f 20 64 6
9 73 20 6
9 7
3 2	0 6		4 61 7 7
6   
7>
