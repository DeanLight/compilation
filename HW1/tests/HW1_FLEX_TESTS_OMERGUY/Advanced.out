1 COMMENT %% My comment brings all the boys to the yard
2 COMMENT % it's all about the % bout da % ba dam tzzzz%%%%
5 STREAM 
8 STREAM 
9 COMMENT % =====================================================
10 COMMENT %					where is my dog
11 COMMENT % =====================================================
14 STREAM stream
23 STREAM stream
stream
stream
can't find him
stream
stream
30 STREAM momma
just killed a snail
put a salt against his head
pour the flour now he's bread
34 STREAM endstreamadafucka
38 STRING streamendstream
40 STRING stream

endstream
44 STREAM (his name is fluffy)
47 STRING            __.                                              
        .-".'                      .--.            _..._    
      .' .'                     .'    \       .-""  __ ""-. 
     /  /                     .'       : --..:__.-""  ""-. \
    :  :                     /         ;.d$$    sbp_.-""-:_:
    ;  :                    : ._       :P .-.   ,"TP        
    :   \                    \  T--...-; : d$b  :d$b        
     \   `.                   \  `..'    ; $ $  ;$ $        
      `.   "-.                 ).        : T$P  :T$P        
        \..---^..             /           `-'    `._`._     
       .'        "-.       .-"                     T$$$b    
      /             "-._.-"               ._        '^' ;   
     :                                    \.`.         /    
     ;                                -.   \`."-._.-'-'     
    :                                 .'\   \
48 STRING Here is the second half :'(
49 STRING  \ \ \         
    ;  ;                             /:  \   \ \ . ;        
   :   :                            ,  ;  `.  `.;  :        
   ;    \        ;                     ;    "-._:  ;        
  :      `.      :                     :         \/         
  ;       /"-.    ;                    :                    
 :       /    "-. :                  : ;                    
 :     .'        T-;                 ; ;        
 ;    :          ; ;                /  :        
 ;    ;          : :              .'    ;       
:    :            ;:         _..-"\     :       
:     \           : ;       /      \     ;      
;    . '.         '-;      /        ;    :      
;  \  ; :           :     :         :    '-.      
'.._L.:-'           :     ;         ;    . `. 
                     ;    :          :  \  ; :  
                     :    '-..       '.._L.:-'  
                      ;     , `.                
                      :   \  ; :                
                      '..__L.:-'
51 STRING Found him. Do you know why ? because
52 STRING 
 __   __.___  ___.    .______        ___   .___________..___  ___.      ___      .__   __. 
|  | (_ )   \/   |    |   _  \      /   \  |           ||   \/   |     /   \     |  \ |  | 
|  |  |/|  \  /  |    |  |_)  |    /  ^  \ `---|  |----`|  \  /  |    /  ^  \    |   \|  | 
|  |    |  |\/|  |    |   _  <    /  /_\  \    |  |     |  |\/|  |   /  /_\  \   |  . `  | 
|  |    |  |  |  |    |  |_)  |  /  _____  \   |  |     |  |  |  |  /  _____  \  |  |\   | 
|__|    |__|  |__|    |______/  /__/     \__\  |__|     |__|  |__| /__/     \__\ |__| \__|

60 STRING Hello dis is dawg
61 EOF 
